css/style.css

```diff
@@ -71,7 +71,8 @@ body {
   display: block;
 }
 
-/*Hero section*/
+/* Hero section */
+
 #main {
   margin-top: 70px;
```

Comment:
- [ ] [REQUIRED] Please, make sure that the website is the same as the Figma design. The `main` section has a background image, please add it.

Figma Design | Your Design
-------------|-------------
![image](https://user-images.githubusercontent.com/60979926/170319871-b0b226c3-3132-479a-a098-f5e65108a72f.png) | ![image](https://user-images.githubusercontent.com/60979926/170318191-a89a618c-cb1e-4d2e-961c-362fa6daf347.png)